charter.md

```diff
@@ -0,0 +1,34 @@
+#Charter for Working Group
+
+There are many situations in which it is desirable to share a digital credential with another person. For example, you may want to provide access to your vehicle to a Valet. You may also want to provide access to your home to your cat sitter. Or, you may want to share a hotel key with your spouse. Today, no such standardized method exists in a cross-platform, multidisciplinary capacity. 
+
+The WG charter includes the definition and standardization of a protocol that will facilitate such credential transfers from individual to individual. The protocol will leverage a “relay server” to transfer data from sender to recipient. The scope of the transfer is limited to a single origin device and a single destination device.
+
+Privacy goals include:
+
+- The relay server should not see sensitive details of the share
+- The relay server should not be able to provision the credential itself, acting as an intermediary for the recipient (MiTM)
+- The relay server should not know the identity of the sender nor receiver
+- Device OEMs should not be able to identity the social graph of whom has shared with whom
+
+Sufficient security measures should be embedded in the protocol in an effort to:
+
+- Ensure only the intended recipient is able to provision the credential
+- Ensure the credential can only be provisioned once (Anti-replay)
+- Ensure the sender has intent to share (secure user intent)
+- Allow the sender to revoke the credential, either before or after the recipient has accepted, where applicable
+
+The solution the WG comes up with must:
+
+- Allow a sender to initiate a share
+- Allow a recipient to view the share request, and provision the credential associated with the share upon receipt
+
+Out of scope topics for the proposed WG are:
+
+- Defining the mechanism the receiver will use (which APIs) in order to provision the credential
+- The WG will define the full set of different credential types that could be shared. A subset of these credential types adhere to a public standard. For these credential types, the format of the Provisioning Information shall be defined by the appropriate standard. Other credential types may be proprietary. For these credential types, the protocol the WG aims to establish shall not define the actual format nor content of each field within the Provisioning Information.
+- The User Interface (UI) that is displayed to the sender or receiver during sending or receiving - this will depend on the device OEM’s UI and HI guidelines.
+
+The WG will deliver a set of APIs and workflows to facilitate secure credential transfer. The WG must consider all Privacy and Security considerations in an effort to perform the credential transfer in a secure manner. Privacy-preserving algorithms such as field-level encryption will be used to protect data in transit.
+
+The starting point for Secure Credential Transfer WG discussions shall be draft-secure-credential-transfer.
```

Comment:
I'm a bit confused by bullet points that say "WG will" in Out of scope topics.

Does it mean that WG will not do any of the things mentioned here or it will?

Reply:
The last 2 paragraphs are a conclusion, rather than a part of "Out of scope" topic.
Sorry for confusion.